semantics/executable-spec/src/Control/Iterate/SetAlgebraInternal.hs

```diff
@@ -999,6 +999,7 @@ compute:: Exp t -> t
 compute (Base rep relation) = relation
 
 compute (Dom (Base SetR rel)) = rel
+compute (Dom (Base MapR x)) = Sett (Map.keysSet x)
```

Comment:
This reminds me of #1775, but doesn't help (yet?) because `STS.bbodySlots = eval (dom nesOsched)` forces the evaluation when creating the `BbodyEnv`: https://github.com/input-output-hk/cardano-ledger-specs/blob/d0f7eded325c4e4b36e44b4b49fd75b0c83f00f3/shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/API/Validation.hs#L72